app/services/qa/linked_data/mapper/context_mapper_service.rb

```diff
@@ -21,7 +21,7 @@ class << self
           def map_context(graph:, context_map:, subject_uri:)
             context = []
             context_map.properties.each do |property_map|
-              values = property_map.values(graph, subject_uri)
+              values = values(property_map, graph, subject_uri)
```

Comment:
Extracted to a private method to appease rubocop.

Reply:
@elrayle you may be sad later on when you try and find values.  It is not a very distinct name. Is it really `expanded_values`?

Reply:
Addressed by renaming method `values` to `property_values`.